portfolio.html

```diff
@@ -0,0 +1,102 @@
+<html lang="en">
+<head>
+<title>Tom Hughes | Portfolio</title>
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+<link rel="stylesheet" type="text/css" href="normalize.css">	
+<link rel="stylesheet" type="text/css" href="portfoliostyle.css"> 
+<script src="https://kit.fontawesome.com/05d75d114c.js" crossorigin="anonymous"></script>
+</head>	
+<body>
+	<nav>
+		<div class="anchors">
+			<a class ="nav_link" href="#about">About</a>
+			<a class ="nav_link" href="#portfolio">Portfolio</a>
+			 <a class ="nav_link" href="#socials">Socials</a>
+		</div>
+	</nav>
+	<header>
+		<div>
+			<h2>Welcome to</h2>
```

Comment:
this should not be a `h2` because its purpose on the page is not as a sub-heading, rather it's just for style. so you should use either `p` or `span` or something